README.md

```diff
@@ -62,11 +62,12 @@
     <img src="https://img.shields.io/badge/benchmarks-asv-lightgrey" />
     </a>
   </td>
-  </tr>
+</tr>
+<tr>
   <td>Publications</td>
   <td>
-    <a href="https://doi.org/10.5281/zenodo.3762635">
-    <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.3762635.svg" alt="zenodo reference">
+    <a href="https://doi.org/10.5281/zenodo.593284">
+    <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.593284.svg" alt="zenodo reference">
```

Comment:
This DOI is the special one that always resolves to the latest version; I figure that was the best option here since this URL isn't really shown to the user anyway.